set.mm

```diff
@@ -6908,8 +6908,8 @@ use disjunction (although this is not required since definitions are
      It was proved by Leibniz, and it evidently pleased him enough to call it
      _praeclarum theorema_ (splendid theorem).  (Contributed by NM,
      12-Aug-1993.)  (Proof shortened by Wolf Lammen, 7-Apr-2013.) $)
-  prth $p |- ( ( ( ph -> ps ) /\ ( ch -> th ) )
-       -> ( ( ph /\ ch ) -> ( ps /\ th ) ) ) $=
+  anim12 $p |- ( ( ( ph -> ps ) /\ ( ch -> th ) )
+              -> ( ( ph /\ ch ) -> ( ps /\ th ) ) ) $=
     ( wi id im2anan9 ) ABEZABCDEZCDHFIFG $.
```

Comment:
Shall we really replace the "historical" name praeclarum theorema (~prth in short) by the technical name ~anim12? Some time ago, I suggested to replace the names of some "stoic" theorems by  their technical counterparts, but this was strongly rejected by @david-a-wheeler I think. Furthermore, it is exercised practice in set.mm to use a corresponding label if the (mostly famous) theorem has a special name (e.g., ~cramer for Cramer's rule, and not ~sollineqmat or something similar).

Reply:
It is not obvious that the label "prth" stands for "praeclarum theorema". And then, what that "brilliant theorem" could be is not clear (Leibniz could receive a "poorest naming ever" for this, by the way). On the other hand, you could guess the statement of "anim12" just by its label.

Actually, here is how I discovered this problem: I needed "anim12" in a proof, so I typed "ASSIGN LAST anim12" but that was rejected by MM, so I went to add it to set.mm, and then, minimizing its proof, I realized that it was already in set.mm with label "prth". I had to look at its comment to understand why it was thus labeled.

Reply:
Merging after two days, for fear of further merge conflicts.
To clarify what I wrote above: the name "praeclarum theorema" should stay in the comment, since it somehow stuck, but the label "prth" does not really hint at that name (unless we already know it).